cluster-http/src/main/resources/reference.conf

```diff
@@ -7,12 +7,21 @@
 
 akka.management {
 
-  # registers bootstrap routes to be included in akka-management's http endpoint
+  # registers http management to be included in akka-management's http endpoint
   http.route-providers += "akka.management.cluster.ClusterHttpManagement"
 
+  # registers cluster healthchec to be included in akka-management's http endpoint
+  http.route-providers += "akka.management.cluster.ClusterHealthCheck"
+
   cluster.http {
     # settings for the cluster routes
     # endpoint details are to be configured in akka-management
-  }
 
+    healthcheck {
+      # Ready health check returns 200 when cluster membership is in the following states.
+      # Intended to be used to indicate this node is ready for user traffic so Up/WeaklyUp
+      # Valid values: "Joining", "WeaklyUp", "Up", "Leaving", "Exiting", "Down", "Removed"
+      ready-states = ["Up", "WeaklyUp"]
```

Comment:
Nice to make this configurable

Reply:
I didn't fancy the Weakly Up argument :)